src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -333,7 +325,7 @@ class JesBackend extends Backend with LazyLogging {
 
   // Create an input parameter containing the path to this authentication file
   def gcsAuthParameter(descriptor: WorkflowDescriptor): Option[JesInput] = {
-    if (JesConf.authMode == RefreshTokenMode || JesConf.dockerCredentials.isDefined) {
+    if (JesConf.localizeWithRefreshToken || JesConf.isDockerAuthenticated) {
       authGcsCredentialsPath(Option(gcsAuthFilePath(descriptor)))
```

Comment:
Since `authGcsCredentialsPath` is only one line, maybe it's better to just inline it